capitalizeAndFilter.test.js

```diff
@@ -0,0 +1,14 @@
+import capitalizeAndFilter from './capitalizeAndFilter.js';
+
+describe('takes in an array of strings and filters out strings starting with the letter F/f', () => {
+  it('should remove strings starting with the F/f', () => {
+    const array = ['leprachaun', 'frog', 'freedom', 'speech'];
+    const filter = capitalizeAndFilter(array);
+    
+    const secondArray = ['frisbee', 'golf', 'longboarding', 'Football'];
+    const secondFilter = capitalizeAndFilter(secondArray);
+
+    expect(filter).toEqual(['leprachaun', 'speech']);
```

Comment:
Remember that this function is supposed to capitalize the strings (first letter in the string) as well as filter. So the you should be expecting "Leprachaun" and "Speech" to come back from your test.